data-table/__tests__/table.js

```diff
@@ -20,6 +20,29 @@ describe('basic tests to ensure createDataTable can function well', () => {
         expect(contents.substr(0, 12)).toEqual("+ Add column");
     });
 
+    test("test default row name", () => {
```

Comment:
this is testing a NON-`default` row name, but the explanation says it's testing the `default` row name